plugin/src/main/java/io/jenkins/plugins/casc/impl/configurators/DataBoundConfigurator.java

```diff
@@ -145,10 +145,16 @@ private T tryConstructor(Constructor<T> constructor, Mapping config, Configurati
             for (Object arg : args) {
                 argumentTypes.add(arg != null ? arg.getClass().getName() : "null");
             }
+            List<String> parameterTypes = new ArrayList<>(parameters.length);
+            for (Parameter parameter: parameters){
+                parameterTypes.add(parameter.getParameterizedType().getTypeName());
+            }
             throw new ConfiguratorException(this,
                     "Failed to construct instance of " + target +
                             ".\n Constructor: " + constructor.toString() +
-                            ".\n Arguments: " + argumentTypes, ex);
+                            ".\n Arguments: " + argumentTypes +
+                            ".\n Expected ParamNames: " + names +
```

Comment:
```suggestion
                            ".\n Expected names of parameters: " + String.join(", ", names) +
```

Reply:
Done :+1:

Reply:
They don't seem to be pushed 😓 Also pushing to mark them as resolved 😆

Reply:
Yeah thought of pushing in one single commit :)